src/controller/org.controller/index.js

```diff
@@ -628,14 +628,14 @@ router.put('/org/:shortname/user/:username',
   /*
   #swagger.tags = ['Users']
    #swagger.operationId = 'userUpdateSingle'
-  #swagger.summary = "Updates a user entry for the specified username and organization shortname (accessible to all registered users)"
+  #swagger.summary = "Updates information about a user for the specified username and organization shortname (accessible to all registered users)"
   #swagger.description = "
         <h2>Access Control</h2>
         <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Regular User:</b> Updates the user's own user entry. Only name fields may be changed.</p>
-        <p><b>Admin User:</b> Updates a user entry for users in the Admin's organization. Allowed to change all fields except org_short_name. </p>
-        <p><b>Secretariat:</b> Updates a user entry in any organization. Allowed to change all fields.</p>"
+        <p><b>Regular User:</b> Updates the user's own information. Only name fields may be changed.</p>
+        <p><b>Admin User:</b> Updates information about a user for users in the Admin's organization. Allowed to change all fields except org_short_name. </p>
```

Comment:
This is a nitpick, but I think this should be "Updates information about any user in the Admin's organization."